save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/projectcollection/CollectionView.kt

```diff
@@ -79,10 +109,60 @@ class CollectionView : AbstractView<CreationViewProps, State>() {
                 }
             }
         },
+        isTransparentGrid = true,
         initialPageSize = 10,
         useServerPaging = false,
         usePageSelection = false,
     )
+    private val privateProjectsTable: FC<TableProps<ProjectDto>> = tableComponent(
+        columns = {
+            columns {
+                column(id = "organization", header = "Organization", { organizationName }) { cellContext ->
+                    Fragment.create {
+                        td {
+                            a {
+                                href = "#/${cellContext.row.original.organizationName}"
```

Comment:
We don't use `a` tag, do we? Now we have `Link`, `navigateTo` and some other variants. Please check and change it

Reply:
Fixed